sunpy/data/data_manager/cache.py

```diff
@@ -155,18 +155,24 @@ def _download_and_hash(self, urls):
             Path, hash and URL of the file.
         """
         def download(url):
-            path = self._cache_dir / get_filename(urlopen(url), url)
+            # Disable SSL check for this download.
```

Comment:
:eyes: for why?

Reply:
SSL error was causing urlopen to fail. 
But it fails to download the file regardless.

Reply:
We shouldn't do this here anyway, as it's going to apply to all users of the cache in all situations?

Reply:
Removed. I guess we can just hope the server recovers instead.